app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -251,17 +251,17 @@ class SettingsFragment : PreferenceFragmentCompat(), AccountObserver {
         val makeDefaultBrowserKey = getPreferenceKey(pref_key_make_default_browser)
         val leakKey = getPreferenceKey(pref_key_leakcanary)
         val debuggingKey = getPreferenceKey(pref_key_remote_debugging)
-        val preferenceAlwaysOpenInPrivateModeKey = getPreferenceKey(
-            pref_key_launch_links_in_private_mode
+        val preferenceLaunchLinksPrivateTabKey = getPreferenceKey(
```

Comment:
nit: preferenceLaunchLinks*In*PrivateTabKey?

Reply:
I'm changing this key in an PR today and will update this key :)